### PR TITLE
Added package files & updated package description

### PR DESCRIPTION
**Pull request by anniejohansson**

### CODE REVIEW

File: WhatsInSeason/DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Package: WhatsInSeason
+Type: Package
+Title: Tells You the Seasonal Fruit & Veg in Your Region
+Version: 0.1.0
+Author: Annie Johansson
+Maintainer: Annie Johansson  <annie.johansson@student.uva.nl>
+Description: One step to becoming more environmentally friendly is to consume local produce that is in season. 
+    This package informs about what fruits and vegatables are in season in different countries around Europe.
```
Comment (millapihlajamaki) [inline WhatsInSeason/DESCRIPTION:8]: Sounds like a really useful package, and the description is clear and informative.

File: WhatsInSeason/R/in_season_test.R [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## A basic prototype to test what the WhatsInSeason package will do.
```
Comment (millapihlajamaki) [inline WhatsInSeason/R/in_season_test.R:1]: Nice example :)

File: WhatsInSeason/README.Rmd [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Here is a simple example: 
+```{r, include=FALSE}
+source("R/in_season_test.R")
+```
+
+```{r}
+whatsinseason("summer", "Netherlands")
+```
+
+### Ideas for further functions that can be built into this package:
```
Comment (millapihlajamaki) [inline WhatsInSeason/README.Rmd]: I'm wondering if you could also expand the base function to allow the user to specify the place and season in different formats? Some countries might, for example, have variation in seasonal products due to area-wise variation in weather/soil conditions, so specifying the entire country might be misleading.

File: WhatsInSeason/DESCRIPTION (hunk 1 above)
Comment (lucasrosenbusch) [reply WhatsInSeason/DESCRIPTION:8]: I agree! I also like the idea. Maybe, you can also generate an entire season calendar for the year.

File: WhatsInSeason/R/in_season_test.R (hunk 2 above)
Comment (lucasrosenbusch) [reply WhatsInSeason/R/in_season_test.R:1]: Great idea to already come up with this example!

**Pull request merged**